pkg/granted/registry/cfregistry/cfregistry.go

```diff
@@ -37,11 +37,16 @@ func (r *Registry) getClient() (awsv1alpha1connect.ProfileRegistryServiceClient,
 	if r.client != nil {
 		return r.client, nil
 	}
-	cfg, err := config.LoadDefault(context.Background())
+
+	err := config.SwitchContext(r.opts.Name)
```

Comment:
This needs to be removed from this PR and put into a separate PR, it will break the existing behaviour because users have a `default` key for the CF file context, it doesn't match the profile registry